parser/pkgpath.go

```diff
@@ -8,9 +8,10 @@ import (
 	"os/exec"
 	"path"
 	"path/filepath"
-	"strconv"
 	"strings"
 	"sync"
+
+	"golang.org/x/mod/modfile"
```

Comment:
I'm not sure if in this particular case we want to introduce third-party dependency. We can just copy single function from `golang.org/x/mod/modfile`. Tell me plz which option do you prefer.

Reply:
i prefer to have zero dependencies.

Reply:
Updated